### ISSUE TITLE
Timestamps aren't serialized correctly in json

### ISSUE THREAD

**Issue by lawrencejones** [lawrencejones/pgsink#3]
```json
{
  "created_at": "\u0000\u0001\ufffd\ufffd\ufffd־\ufffd"
}
```

**Comment by lawrencejones** [comment]
<img width="123" alt="Screenshot 2020-06-29 at 20 33 54" src="https://user-images.githubusercontent.com/3518874/86048136-e5dafd80-ba47-11ea-84e6-89b0340306cf.png">

**Comment by lawrencejones** [comment]
Fixed by #142
